apps/api/v2/src/modules/organizations/inputs/create-organization-user.input.ts

```diff
@@ -0,0 +1,17 @@
+import { CreateUserInput } from "@/modules/users/inputs/create-user.input";
+import { MembershipRole } from "@prisma/client";
+import { IsString, IsOptional, IsBoolean, IsEnum } from "class-validator";
+
+export class CreateOrganizationUserInput extends CreateUserInput {
```

Comment:
I created a base `User` DTO that we can then expand on for org users. The base `CreateUserInput` was taken from V1 API `apps/api/v1/lib/validations/user.ts`